### PR TITLE
feat: Adds types to Britecharts via declaration files

### PR DESCRIPTION
**Pull request by Golodhros**
## Description
Adds declaration files to Britecharts
Working on our test project: https://github.com/britecharts/britecharts-typescript-test-project

## Motivation and Context
Adding Types to Britecharts directly

## How Has This Been Tested?
Running yarn start:dev on the test project and using Yarn link on development

## Types of changes

-   [ ] Refactor (changes the way we code something without changing its functionality)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [ ] Latest master code has been merged into this branch
-   [ ] No commented out code (if required, place // TODO above with explanation)
-   [ ] No linting issues
-   [ ] Build is successful
-   [ ] Code follows the [API Guidelines](http://britecharts.github.io/britecharts/topics-index.html#toc5__anchor)
-   [ ] Updated the documentation
-   [ ] Added tests to cover changes
-   [ ] All new and existing tests passed

Thanks @ImADrafter for your help pushing this forward!!

### CODE REVIEW

File: src/typings/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export * from './charts/legend-component';
+export * from './charts/line-chart';
+export * from './charts/mini-tooltip';
+export * from './charts/scatter-plot';
+export * from './charts/sparkline-chart';
+export * from './charts/stacked-area';
+export * from './charts/stacked-bar-chart';
+export * from './charts/tooltip';
+export * from './helpers/colors';
+export * from './helpers/constants';
```
Comment (gagoar) [inline src/typings/index.d.ts:33]: why do you need to import first?

Comment (DalerAsrorov) [reply src/typings/index.d.ts:33]: I am also curious what's the point of this

Comment (Golodhros) [reply src/typings/index.d.ts]: It seems it wasn't necessary!

**Review by DalerAsrorov — APPROVED** [review]
LGTM! Have you tried using the VS code extension and see how it works? The auto suggestions should work well! This is awesome

**Review by ImADrafter — APPROVED** [review]
So the "declare" was no longer needed as I see, which makes sense since this has nothing to do with module augmentation. Great work @Golodhros !

**Comment by Golodhros** [comment]
> LGTM! Have you tried using the VS code extension and see how it works? The auto suggestions should work well! This is awesome

Yeah, it shows up the comments on the tooltip!

**Comment by Golodhros** [comment]
> So the "declare" was no longer needed as I see, which makes sense since this has nothing to do with module augmentation. Great work @Golodhros !

Thanks, but it was you who unblocked me here! Thank you!

**Pull request merged**